test/integration/targets/docker_volume_facts/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+- block:
+  - name: Create random volume name
+    set_fact:
+      cname: "{{ 'ansible-test-%0x' % ((2**32) | random) }}"
+
+  - name: Make sure volume is not there
+    docker_volume:
+      name: "{{ cname }}"
+      state: absent
+      force: yes
+
+  - name: Inspect a non-present volume
+    docker_volume_facts:
+      name: "{{ cname }}"
+    register: result
+
+  - assert:
+      that:
+      - "not result.exists"
+      - "'docker_volume' in result"
+      - "not result.docker_volume"
```

Comment:
Per your guidance in `docker_network_facts`, this should explicitly test for `none` if it's expecting the result to be `None`.

Reply:
Should I go back and standardize `docker_network_facts` to return an empty dict instead of None to match these? I feel like we should be consistent.

Reply:
No, actually both docker_volume_facts and docker_container_facts returned `None`, but it was documented wrongly. I've now fixed that.